test/test.js

```diff
@@ -27,7 +27,7 @@ describe('Apostrophe cache implementation in redis', function() {
     });
   });
   it('initializes a redis client', function() {
-    assert(apos.caches.client);
+    assert(apos.caches);
```

Comment:
I don't understand this change.  This property should exist at this point still.

Reply:
The test was failing even before making changes. I thought it was an old mistake not fixed.